pom.xml

```diff
@@ -119,6 +120,16 @@
       <artifactId>jsr305</artifactId>
       <version>3.0.1</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
```

Comment:
I'm not sure how it works with OSGi, but the protobuf classes are shaded with sonarlint-core, so maybe can be reused?